application/Modules/Quotes/Models/Quote.php

```
@@ -305,16 +348,24 @@ public function get_date_due($quote_date_created)
+     *
+     * Legacy migration info:
+     * @legacy-file application/modules/quotes/models/Mdl_quote.php
+     * @legacy-function get_quote_number()
      */
     public function get_quote_number($invoice_group_id)
     {
-        $this->load->model('invoice_groups/mdl_invoice_groups');
+        $this->load->model('invoice_groups/invoice_group');
 
         return $this->mdl_invoice_groups->generate_invoice_number($invoice_group_id);
     }
```

PHPMD (2.15.0)

The method get_quote_number is not named in camelCase. (undefined)

(CamelCaseMethodName)